### ISSUE TITLE
Hero Video: Links are incorrectly coded as buttons - 4.1.2 Name, Role, Value 

### ISSUE THREAD

**Issue by syarwick** [stephenravenscraft/loyola-template-4.0#72]
The group of links on top of the video are coded as buttons, but should be coded as links.

Include 'role="link"' on the buttons to ensure assistive technology accurately describes the component

**Comment by stephenravenscraft** [comment]
This has been updated. Please review.

`
                    <div class="grid-wrapper-min-max three-columns">
                        
                        <div class="grid-item">
                            <button role="link" onclick="window.open('https://www.luc.edu','_blank')">
                                Apply now
                            </button>
                        </div>
                        
                        <div class="grid-item">
                            <button role="link" onclick="window.open('https://www.luc.edu','_blank')">
                                Visit campus
                            </button>
                        </div>
                        
                        <div class="grid-item">
                            <button role="link" onclick="window.open('https://www.luc.edu','_blank')">
                                Request info
                            </button>
                        </div>
                    </div>
`
